examples/ram_test_deploys.py

```diff
@@ -0,0 +1,18 @@
+"""Example: Get all invalid remote access and test them"""
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+INVALID_REMOTE_ACCESS_IDS = []
+for remote_access_item in CLIENT.remote_accesses():
```

Comment:
INVALID_REMOTE_ACCESS_IDS = [str(item.id) for t in CLIENT.remote_accesses() if item.is_valid is False]